Blog-projects/Widget-window-in-Qt-Quick-app/fontsbackend.cpp

```diff
@@ -0,0 +1,13 @@
+#include "fontsbackend.h"
+
+#include <QFontDatabase>
+
+FontsBackend::FontsBackend(QObject *parent)
+    : QObject{parent}
+{
+}
+
+QStringList FontsBackend::fontList()
```

Comment:
const method (or even static, if that works for QML too)

Reply:
Done. Set static const.